### PR TITLE
crnnFP16

### PR DESCRIPTION
**Pull request by Charles-258**
add two crnn quantized models

### CODE REVIEW

File: models/text_recognition_crnn/README.md [hunk 1]
```diff
@@ -2,11 +2,23 @@
...
 
+Results of accuracy evaluation with [tools/eval](../../tools/eval).
+
+| Models | Accuracy(%) |
+| ------ | -------------- |
+| CRNN_EN | 81.66 |
+| CRNN_EN_FP16 | 82.01 |
+| CRNN_CH | 71.28 |
+| CRNN_CH_FP16 | 78.63 |
+\*: 'FP16' stands for 'model quantized into FP16'.
```
Comment (zihaomu) [inline models/text_recognition_crnn/README.md:13]: We should carry experiment on more OCR datasets. Then, I think we can merge it. Please add info about the test dataset.

Comment (zihaomu) [reply models/text_recognition_crnn/README.md:13]: @Charles-258, friendly reminder

**Review by zihaomu — APPROVED** [review]
LGTM! 👍

**Comment by fengyuentau** [comment]
We need to add notes in the readme telling users how to use these fp16 models and compare the accuracy with fp32 models' as well.

**Comment by zihaomu** [comment]
@fengyuentau This info can be found in the merged PR #71.

**Comment by fengyuentau** [comment]
Thank you but we still need notes in the readme about how to use these fp16 models with opencv.

**Comment by zihaomu** [comment]
In my opinion, we will have more fp16 models in the future, we need to add a unified note about how to load float16 in OpenCV. How about adding this info in https://github.com/opencv/opencv_zoo/tree/master/benchmark/README.md? 
The main idea is to tell the user that loading float 16 ONNX model requires opencv with this patch [PR in opencv](https://github.com/opencv/opencv/pull/22337).
The best solution for python is `pip install opencv-python-rolling`.

**Pull request merged**